packages/rest/test/unit/rest-server/rest-server.open-api-spec.test.ts

```diff
@@ -29,6 +30,7 @@ describe('RestServer.getApiSpec()', () => {
       basePath: '/api',
       paths: {},
       'x-foo': 'bar',
+      definitions: {},
```

Comment:
Maybe I am too stubborn about this, but cannot we remove `definitions` property from the api spec reported by the server when there were not definitions specified?

See my two comments above for an inspiration.

Reply:
Nope, you're not being stubborn. I just forgot to take those out :)